liiatools/common/converters.py

```diff
@@ -116,8 +119,32 @@ def to_numeric(value, _type, min_value=None, max_value=None, decimal_places=None
     :param min_value: Minimum value allowed
     :param max_value: Maximum value allowed
     :param decimal_places: The number of decimal places to apply
+    :param age: Specifies whether the value needs age specific conversion
     :return: Either a numeric value or a blank string
     """
+    if age == True:
+        # Handle certain instances of strings to convert to age in years
+        try:
```

Comment:
I would consider converting value to string here just in case it's read is as an integer (which would result in a error)